frontend/src/ui/metadata.tsx

```diff
@@ -52,3 +56,48 @@ export const Description: React.FC<DescriptionProps> = ({ text, lines = 2 }) =>
         }}>{text}</div>;
     }
 };
+
+
+type DescriptionProps = {
+    text: string | null;
+    className?: string;
+};
+
+/**
+ * Display an event's or series' description. It is somewhat rendered into
```

Comment:
Similarly here. Maybe just say concretely what transformation is applied. No need to even mention Markdown.

Reply:
I adjusted it but didn't go into details... don't need to exactly describe what the body does.